arch/sim/src/nuttx-names.in

```diff
@@ -32,57 +32,192 @@
+NXSYMBOLS(clearenv)
+NXSYMBOLS(clock)
 NXSYMBOLS(clock_gettime)
+NXSYMBOLS(clock_settime)
 NXSYMBOLS(close)
 NXSYMBOLS(closedir)
 NXSYMBOLS(connect)
+NXSYMBOLS(dlclose)
+NXSYMBOLS(dlerror)
+NXSYMBOLS(dlopen)
+NXSYMBOLS(dlsym)
+NXSYMBOLS(dlsymtab)
 NXSYMBOLS(dup)
+NXSYMBOLS(dup2)
 NXSYMBOLS(exit)
+NXSYMBOLS(fclose)
+NXSYMBOLS(fcntl)
+NXSYMBOLS(fdopen)
+NXSYMBOLS(fgetc)
+NXSYMBOLS(fopen)
+NXSYMBOLS(fpathconf)
+NXSYMBOLS(fprintf)
+NXSYMBOLS(fputc)
+NXSYMBOLS(fputs)
+NXSYMBOLS(fread)
+NXSYMBOLS(free)
+NXSYMBOLS(fseek)
 NXSYMBOLS(fstat)
 NXSYMBOLS(fsync)
+NXSYMBOLS(ftell)
 NXSYMBOLS(ftruncate)
+NXSYMBOLS(ftw)
+NXSYMBOLS(fwrite)
+NXSYMBOLS(getchar)
+NXSYMBOLS(getenv)
+NXSYMBOLS(getopt)
+NXSYMBOLS(getpid)
+NXSYMBOLS(gettimeofday)
+NXSYMBOLS(htons)
+NXSYMBOLS(if_indextoname)
 NXSYMBOLS(if_nametoindex)
 NXSYMBOLS(ioctl)
+NXSYMBOLS(isatty)
+NXSYMBOLS(kill)
+NXSYMBOLS(link)
+NXSYMBOLS(listen)
 NXSYMBOLS(lseek)
+NXSYMBOLS(mallinfo)
+NXSYMBOLS(malloc)
+NXSYMBOLS(malloc_init)
+NXSYMBOLS(memcmp)
+NXSYMBOLS(memcpy)
+NXSYMBOLS(memset)
 NXSYMBOLS(mkdir)
+NXSYMBOLS(mkfifo)
+NXSYMBOLS(mktime)
 NXSYMBOLS(mmap)
+NXSYMBOLS(mount)
+NXSYMBOLS(mq_close)
 NXSYMBOLS(munmap)
+NXSYMBOLS(nanosleep)
+NXSYMBOLS(nftw)
+NXSYMBOLS(ntohs)
 NXSYMBOLS(open)
 NXSYMBOLS(opendir)
+NXSYMBOLS(optarg)
+NXSYMBOLS(optind)
+NXSYMBOLS(optopt)
+NXSYMBOLS(pathconf)
+NXSYMBOLS(perror)
+NXSYMBOLS(pipe)
 NXSYMBOLS(poll)
+NXSYMBOLS(printf)
 NXSYMBOLS(pthread_create)
 NXSYMBOLS(pthread_cond_destroy)
 NXSYMBOLS(pthread_cond_init)
 NXSYMBOLS(pthread_cond_signal)
 NXSYMBOLS(pthread_cond_wait)
+NXSYMBOLS(pthread_detach)
+NXSYMBOLS(pthread_exit)
 NXSYMBOLS(pthread_getspecific)
+NXSYMBOLS(pthread_join)
 NXSYMBOLS(pthread_key_create)
 NXSYMBOLS(pthread_kill)
 NXSYMBOLS(pthread_mutex_destroy)
 NXSYMBOLS(pthread_mutex_init)
 NXSYMBOLS(pthread_mutex_lock)
 NXSYMBOLS(pthread_mutex_unlock)
+NXSYMBOLS(pthread_setcancelstate)
+NXSYMBOLS(pthread_setcanceltype)
 NXSYMBOLS(pthread_setspecific)
 NXSYMBOLS(pthread_sigmask)
+NXSYMBOLS(pthread_testcancel)
+NXSYMBOLS(pthread_yield)
+NXSYMBOLS(ptsname)
+NXSYMBOLS(ptsname_r)
+NXSYMBOLS(putenv)
+NXSYMBOLS(puts)
 NXSYMBOLS(read)
 NXSYMBOLS(readdir)
+NXSYMBOLS(readlink)
+NXSYMBOLS(realloc)
+NXSYMBOLS(recv)
+NXSYMBOLS(recvfrom)
 NXSYMBOLS(rename)
+NXSYMBOLS(rewind)
 NXSYMBOLS(rewinddir)
 NXSYMBOLS(rmdir)
+NXSYMBOLS(sched_setparam)
 NXSYMBOLS(sched_yield)
+NXSYMBOLS(seekdir)
 NXSYMBOLS(select)
+NXSYMBOLS(sem_destroy)
+NXSYMBOLS(sem_getvalue)
+NXSYMBOLS(sem_init)
+NXSYMBOLS(sem_post)
+NXSYMBOLS(sem_timedwait)
+NXSYMBOLS(sem_trywait)
+NXSYMBOLS(sem_wait)
+NXSYMBOLS(send)
+NXSYMBOLS(sendto)
+NXSYMBOLS(setenv)
+NXSYMBOLS(setlogmask)
+NXSYMBOLS(setsockopt)
+NXSYMBOLS(setvbuf)
 NXSYMBOLS(shutdown)
 NXSYMBOLS(sigaction)
 NXSYMBOLS(sigaddset)
+NXSYMBOLS(sigdelset)
 NXSYMBOLS(sigemptyset)
 NXSYMBOLS(sigfillset)
+NXSYMBOLS(sighold)
+NXSYMBOLS(sigismember)
+NXSYMBOLS(sigprocmask)
+NXSYMBOLS(sigrelse)
+NXSYMBOLS(sigtimedwait)
 NXSYMBOLS(sleep)
+NXSYMBOLS(snprintf)
 NXSYMBOLS(socket)
```

Comment:
@yamt, But NuttX's socket is renamed to NXscoket here, we need find why vpnkit_connect call into NXsocket first.

Reply:
-unexported_symbols_list is used on macOS. ie. it isn't renamed to NXsocket.

Reply:
Ok, we are looking the better approach to fix this issue, please wait a moment.

Reply:
sure!